packages/bsky/src/api/app/bsky/actor/getSuggestions.ts

```diff
@@ -1,75 +1,124 @@
+import { mapDefined } from '@atproto/common'
 import AppContext from '../../../../context'
+import { Database } from '../../../../db'
+import { Actor } from '../../../../db/tables/actor'
 import { notSoftDeletedClause } from '../../../../db/util'
 import { Server } from '../../../../lexicon'
+import { QueryParams } from '../../../../lexicon/types/app/bsky/actor/getSuggestions'
+import { createPipeline } from '../../../../pipeline'
+import { ActorInfoMap, ActorService } from '../../../../services/actor'
+import { BlockAndMuteState, GraphService } from '../../../../services/graph'
 
 export default function (server: Server, ctx: AppContext) {
+  const getSuggestions = createPipeline(
+    skeleton,
+    hydration,
+    noBlocks,
```

Comment:
i guess it's diff behavior than we have no, but seems like we should remove mutes form this as well

Reply:
Done!